src/components/POIForm.component.js

```diff
@@ -71,29 +71,26 @@ class POIForm extends Component {
   onSubmit() {}
 
   render() {
+    const stories = this.props.stories
+
     return (
       <Form horizontal>
         <PageHeader>
           POI Input Form
```

Comment:
Might be a little redundant to say "Form" since the user knows they're looking at a form =P

We could say something like "Create POI" instead